test/ci/cpu_integration_tests_linux.yml

```diff
@@ -32,7 +32,7 @@ jobs:
   # Conda environment creation
   - bash: |
       cd /home/vsts/work/1/s
-      python tools/generate_conda_files.py
+      python tools/generate_conda_files.py --python-version 3.6.8
```

Comment:
Seems like it would be good to talk about the strict usage of 3.6.8 in a few places in here - I'll start a discussion in the current thread

Reply:
The conda environment is very strictly tied to 3.6.8 it seems. I had hoped to run with multiple Python versions (and this was the first step in that direction), but such was not to be.

Reply:
Hmm, the generate_conda_files.py already uses 3.6.8. Is it still required here? https://github.com/interpretml/interpret-text/blob/master/tools/generate_conda_files.py

Reply:
When I tried a different Python version, the building of the conda environment from that generated file failed.

Reply:
And I did change the `generate_conda_files.py` script to take the python version from this argument